man/ru/d2df.3.txt

```diff
@@ -70,7 +70,7 @@ sv_master_interval <число>
 
 @h2 Технические параметры
 net_master_list <адрес>
-	На какой адрес мастер-сервера будут посылаться данные. Можно указывать несколько, например, mpms.doom2d.org:25665,deadsoftware.ru:25665
+	На какой адрес мастер-сервера будут посылаться данные. Можно указывать несколько, например, dfms.doom2d.org:25665,dfms2.doom2d.org:1005,terminalcorner.ru:25665'
```

Comment:
Two samples are enough. Leave only `dfms.doom2d.org:25665` and `dfms2.doom2d.org:1005` here for now.